C/services/storage/storage_api.cpp

```diff
@@ -537,7 +599,7 @@ string  responsePayload;
 			respond(response, SimpleWeb::StatusCode::client_error_bad_request, responsePayload);
 		}
 
-		respond(response, responsePayload);
+		//respond(response, responsePayload);
```

Comment:
In the original code, respond was getting called twice in this function, hence HTTP response was being appended twice alongwith "\r\n\r\n" end marker. I tried just fixing this, but it was not sufficient and I was still getting EoF's in HTTP client code.